contracts/SponsorFaucet.sol

```diff
@@ -2,18 +2,35 @@ pragma solidity 0.5.11;
 
 import "@openzeppelin/contracts/ownership/Ownable.sol";
 import "@openzeppelin/contracts/lifecycle/Pausable.sol";
+import "@openzeppelin/contracts/access/roles/WhitelistedRole.sol";
 import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
 import "./Lib/ReentrancyGuard.sol";
 import "./InternalContract.sol";
 
-contract SponsorFaucet is Ownable, Pausable, ReentrancyGuard {
+contract SponsorFaucet is Ownable, Pausable, ReentrancyGuard, WhitelistedRole {
```

Comment:
WhitelistedAdminRole is enough?

Reply:
yeah, change to whitelistAdmin to reduce codesize.